feature-pack/src/main/resources/modules/system/layers/base/org/infinispan/main/module.xml

```diff
@@ -42,7 +42,7 @@
         <module name="org.jboss.jandex"/>
         <module name="org.jboss.logging"/>
         <module name="org.jboss.marshalling"/>
-        <module name="org.jgroups"/>
+        <module name="org.jgroups" optional="true"/>
```

Comment:
My impress from chat is this might not be safe in a situation where the infinispan subsystem is provisioned for local caching but nothing else in the config results in jgroups being provisioned.

Reply:
The infinispan subsystem itself ensures that the jgroups module is provisioned.